### ISSUE TITLE
New Command Idea

### ISSUE THREAD

**Issue by BakaTekku** [quackest/dancetools#11]
Mod has been very useful and fun, great work mate.  Could  weather be added as a command?
usage would be **weather name**
names being: Eclipsed, Flooded, Foggy, Rainy, and Stormy
so we could set weather for all moons by doing weather Stormy

**Comment by sword11247** [comment]
yes i want this but gamemaster and chat commands weather do not work so something might be broken with setting weather

**Comment by quackest** [comment]
I could take a look into it over the weekend. It seems like it could be a pretty neat command. 

Ty for suggestions.

**Comment by sword11247** [comment]
> I could take a look into it over the weekend. It seems like it could be a pretty neat command.
> 
> Ty for suggestions.

i think theres a unused weather called dust clouds
